spec/system/order_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe 'Order - buying a Reward', type: :system do
+  before do
+    driven_by(:rack_test)
+  end
+
+  let(:employee1) { create(:employee) }
+  let(:employee2) { create(:employee) }
+  let!(:kudo) { create(:kudo, receiver: employee1) }
+  let!(:reward1) { create(:reward, price: 1) }
+
+  context 'when I buy a Reward' do
```

Comment:
I recommend _betterspecs_ for specs and context naming convention :) 
```suggestion
  context 'when an employee buys a reward' do
```